src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -41,9 +41,9 @@ def reinit_data(self):
         self.if_alias_map, \
         self.if_id_map, \
         self.oid_sai_map, \
-        self.oid_name_map = Namespace.init_namespace_sync_d_interface_tables(self.db_conn)
+        self.oid_name_map, _, _ = Namespace.init_namespace_sync_d_interface_tables(self.db_conn)
 
-        self.if_bpid_map = Namespace.dbs_get_bridge_port_map(self.db_conn, mibs.ASIC_DB)
+        self.if_bpid_map = Namespace.dbs_get_bridge_port_map(self.db_conn)
```

Comment:
why we don't need second Argument Here ?

Reply:
dbs_get_bridge_port_map() does not use the db name, the function only retrieves data from ASIC_DB, so did not have to use an argument to specify the db name.